src/test/java/org/cactoos/http/io/AutoClosedInputStreamTest.java

```diff
@@ -45,12 +48,13 @@ public void autoClosesTheStream() throws Exception {
         final CloseableInputStream closeable = new CloseableInputStream(
             new DeadInputStream()
         );
-        try (final InputStream ins = new AutoClosedInputStream(closeable)) {
-            int read;
-            do {
-                read = ins.read();
-            } while (read != -1);
-        }
+        new Assertion<>(
+            "must read the stream",
+            new TextOf(
+                new ReadBytes(new AutoClosedInputStream(closeable))
+            ),
+            new TextIs("")
```

Comment:
@victornoel Does it matter for this test whether `Text` of `DeadInputStream` is ""? The test is supposed to assert that the stream is closed after all bytes are read, which is the next assertion. Here I would just write something like `new ReadBytes(new AutoClosedInputStream(closeable)).asBytes()`.